### DISCUSSION TITLE
Transfered domain is still waiting for previous registrar to release it

### DISCUSSION THREAD

**Discussion by phcoliveira** [vercel/community#4999] [Help]
### Summary

Hello there!

I transfered a domain out of Vercel about 5 hours before its expiration. I used the the transfer code normally.

However, the current registrar (Namecheap) is still waiting for the previous registrar to release it. The transfer was initiated on November 23.

Please help me out. What can I do here? My Vercel account is `phcoliveira`.

Thanks!

### Steps to Reproduce

Transfer a domain out hours before its expiration.

**Comment by amyegan** [comment]
Thanks for reaching out about your domain transfer. I know that transferring a domain can take up to a week, and I see a current status of "Transfer Away Pending" on the domain. Because the domain already expired, I'm going to double check with our Domains team just to be sure everything is in order.

**Reply by phcoliveira** [reply]
Thank you very much, @amyegan. I received an email from Namecheap notifying the completion just now.
I thought it was your doing in releasing it, actually.
I will close this ticket as there is nothing else to be done.
Kind regards!

**Reply by amyegan** [reply]
I'm glad to hear it worked! Thanks for letting me know 😄
